tx_check.go

```diff
@@ -250,19 +250,20 @@ func verifyKeyOrder(pgId common.Pgid, pageType string, index int, key []byte, pr
 
 type checkConfig struct {
 	kvStringer KVStringer
-	pageId     uint
+	pageId     uint64
```

Comment:
Please change this in a separate PR, so that we can merge right away

Reply:
Opened a PR: https://github.com/etcd-io/bbolt/pull/699